src/features/UserList/UserList.tsx

```diff
@@ -1,34 +1,51 @@
 import React from 'react';
 import {useQuery} from "react-query";
-import {getAllUsers} from "../common/api/api";
+import {reactTestProjectAPI} from "../common/api/api";
+import {ResponseDataType, UserType} from "../common/types/types";
+import {makeStyles} from '@material-ui/styles';
 
 type UserListProps = {
     title: string
 }
+const useStyles = makeStyles((theme: unknown) => ({
+        root: {
+            margin: 20,
+        },
+        title: {
+            fontWeight: 700,
+            fontSize: 24,
+        },
+        item: {
+            listStyleType: "none",
+            paddingTop: 10,
+        }
+    }
+));
 
 export const UserList: React.FC<UserListProps> = ({title}) => {
 
-    const { isLoading, error, data, isFetching } = useQuery("userList", getAllUsers);
+    const classes = useStyles();
 
-    console.log(data, isLoading)
+    const { isLoading, data} = useQuery<ResponseDataType>("userList", reactTestProjectAPI.getAllUsers);
 
     if (isLoading) {
         return (
             <div>Loading...</div>
         );
     }
 
-    const userList = data?.data.map(({first_name, last_name, id}: any) => {
+    const userList = data?.data.map(({first_name, last_name, id}: UserType) => {
```

Comment:
We should somehow map the snake_case to camelCase, we can write a simple util function that does it for us, or let's use `lodash` and it's `camelCase` helper?